src/CurrencyNetwork.ts

```diff
@@ -56,33 +66,45 @@ export class CurrencyNetwork {
     userAddress: string
   ): Promise<UserOverview> {
     await this._checkAddresses([networkAddress, userAddress])
-    const [ overview, decimals ] = await Promise.all([
+    const [
+      overview,
+      { networkDecimals }
+    ] = await Promise.all([
       this._utils.fetchUrl<UserOverviewRaw>(`networks/${networkAddress}/users/${userAddress}`),
       this.getDecimals(networkAddress)
     ])
     return {
-      balance: this._utils.formatToAmount(overview.balance, decimals),
-      given: this._utils.formatToAmount(overview.given, decimals),
-      received: this._utils.formatToAmount(overview.received, decimals),
-      leftGiven: this._utils.formatToAmount(overview.leftGiven, decimals),
-      leftReceived: this._utils.formatToAmount(overview.leftReceived, decimals)
+      balance: this._utils.formatToAmount(overview.balance, networkDecimals),
+      given: this._utils.formatToAmount(overview.given, networkDecimals),
+      received: this._utils.formatToAmount(overview.received, networkDecimals),
+      leftGiven: this._utils.formatToAmount(overview.leftGiven, networkDecimals),
+      leftReceived: this._utils.formatToAmount(overview.leftReceived, networkDecimals)
     }
   }
 
   /**
-   * Returns the decimals specified in a currency network.
+   * Returns the network decimals and interest decimals specified in a currency network.
    * @param networkAddress Address of currency network.
    * @param decimals If decimals are known they can be provided manually.
    */
-  public async getDecimals (networkAddress: string, decimals?: number): Promise<number> {
+  public async getDecimals (
+    networkAddress: string,
+    decimalsOptions: DecimalsOptions = {}
+    ): Promise<DecimalsObject> {
+    const { networkDecimals, interestRateDecimals } = decimalsOptions
+    const decimalsObject = { networkDecimals, interestRateDecimals }
```

Comment:
Also what was your reason to combine those two and not have two functions getDecimals, getInterestDecimals?

Reply:
I also thought about this alternative. But I wanted to keep the number of requests as small as possible because we retrieve the decimals currently via the relay server. If we have two functions we would need to call the same request twice...

But I also see the point that splitting it up is cleaner. Also `getInterestRateDecimals` would only be needed for `CurrencyNetwork` and `Trustline` related functions :thinking:

Reply:
Makes sense, lets keep it like this for now.